AsyncDisplayKit/Private/ASLayoutTransition.h

```diff
@@ -47,7 +47,7 @@
 /**
  * Insert and remove subnodes that where added or removed between the previousLayout and the pendingLayout
```

Comment:
This doesn't make sense to me...don't added subnodes need to be put in at the beginning, but we have to wait until the end to remove the deleted ones?  How can both of this be handled in one call?  I guess by setting up a completion block?  Still, the premise of the method is confusing.

Is this basically commitTransition, like commitAnimations?